src/vs/workbench/api/browser/mainThreadEditors.ts

```diff
@@ -125,7 +125,7 @@ export class MainThreadTextEditors implements MainThreadTextEditorsShape {
 			// preserve pre 1.38 behaviour to not make group active when preserveFocus: true
 			// but make sure to restore the editor to fix https://github.com/microsoft/vscode/issues/79633
 			activation: options.preserveFocus ? EditorActivation.RESTORE : undefined,
-			ignoreOverrides: true
+			ignoreOverrides: uri.fsPath.toLowerCase().endsWith('ipynb') || uri.fsPath.toLowerCase().endsWith('sql') ? false : true // {{SQL CARBON EDIT}}
```

Comment:
do we know that uri will never be undefined here?

Reply:
I've been testing this myself. I haven't gotten into this case yet, but we should be defensive about this for uri and probably uri.fsPath as well.

Reply:
yeah, particularly with the change coming so close to the release.  unless, we can see a null check further up the callstack, or uri being referenced unchecked lower in the stack (in which case it will fail anyways)

Reply:
right, those 2 scenarios don't appear to be true.

Reply:
there's no null check for uri, but further up the callstack they're resolving the document uri before calling this: https://github.com/microsoft/azuredatastudio/blob/b02b41815d06ac472de81c5cafdcc6e9283fed8a/src/vs/workbench/api/common/extHost.api.impl.ts#L455